pkgs/by-name/sd/SDL2_mixer/package.nix

```diff
@@ -38,11 +40,17 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
   ];
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
-    AudioToolbox
-    AudioUnit
-    CoreServices
-  ];
+  buildInputs =
+    [
+      wavpack
+      libxmp
```

Comment:
`libxmp` does not have a maintainer listed in nixpkgs. SDL2_mixer would be the biggest dependent for it.
How much do you expect libxmp things to break, and might it make sense to adopt it if it now takes some load?

Upstream maintenance seems slow but acceptable. Your PR is fixing actual issues. I will certainly *not* block this over lack of maintenance. But might be relevant to consider.

Seeing as SDL2_mixer doesn't have a maintainer either, it probably doesn't matter anyways...

Reply:
Tbf the only relevant thing pulling SDL2_mixer is the AppImage FHS environment - we should investigate whether that actually needs SDL2_mixer. But that is a different PR.

Reply:
https://github.com/NixOS/nixpkgs/blob/dab7ae6c0729d752ed5527af3715f21380d0e9be/pkgs/build-support/appimage/default.nix#L219-L220